AirportTicketBookingExercise/Logic/Service/UserService.cs

```diff
@@ -0,0 +1,69 @@
+﻿using ATB.Data.Repository;
+using ATB.Data.Models;
+using System.Xml.Linq;
+
+namespace ATB.Logic.Service
+{
+    public class UserService : IUserService
+    {
+        private readonly IUserRepository _userRepository;
+        public UserService(IUserRepository userRepository)
+        { 
+        
+            _userRepository = userRepository;
+        }
+
+        public User? Authenticate(string username, string password, UserType usertype)
+        {
+            var user = GetUserByName(username);
+            if (user != null && user.Password.Equals(password) && user.UserType == usertype) 
+            {
+                return user;
+            }
+            return null;
```

Comment:
same comment about throwing exception. We should throw an unauthenticated exception